src/js-based-plugin/README.md

```diff
@@ -0,0 +1,3 @@
+# JS based plugin / unplugin-ezno
+
+Plugin using Ezno in for Vite, webpack. See the [unplugin repo for details](https://github.com/unjs/unplugin)
```

Comment:
"Ezno in for Vite" should be "Ezno in Vite"

Reply:
Good catch